src/pages/elements/Clusters.js

```diff
@@ -1,15 +1,34 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { Link } from 'react-router-dom';
 import { chunk } from 'lodash';
 import { CardGrid, Row, Col, Card, CardHeading, CardBody, CardFooter, CardTitle } from 'patternfly-react';
 import { sortByName } from '../../components/Utils';
+import SearchBar from '../../components/SearchBar';
 
 function Clusters({ clusters }) {
   // cardsWidth * cardsPerRow must be <= 12 (bootstrap grid)
   const cardWidth = 4;
   const cardsPerRow = 3;
+  const [selected, changeSelected] = useState('Name');
+  const [filterText, changeFilterText] = useState('');
+  const matchedClusters = [];
+  let i;
+  for (i = 0; i < clusters.length; i++) {
+    if (selected === 'Name') {
+      if (clusters[i].name.toLowerCase().indexOf(filterText.toLowerCase()) !== -1) {
+        matchedClusters[matchedClusters.length] = clusters[i];
+      }
+    }
+  }
+  function handleFilterTextChange(txt) {
```

Comment:
perhaps you could avoid declaring `handleFilterTextChange` and `handleSelect` and change the `SearchBar` props to be directly: `handleFilterTextChange={changeFilterText}` and `handleSelect={changeSelected}`